### PR TITLE
fix: minor story accessibility updates to v9 utility examples

### PR DESCRIPTION
**Pull request by smhigley**
This is just some story cleanup to add missing accessible names and some high contrast styles for animations.

Fixes a few ADO bugs.

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/Avatar-mount_base.html">648</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/Avatar-mount.html">660</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/Button-mount_base.html">308</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/Button-mount.html">300</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/Field-mount_base.html">1144</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/Field-mount.html">1113</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/FluentProvider-mount_base.html">727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/FluentProvider-mount.html">727</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/FluentProviderWithTheme-mount_base.html">75</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/FluentProviderWithTheme-mount.html">87</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">40</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">38</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">77</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">79</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/MakeStyles-mount_base.html">849</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/MakeStyles-mount.html">870</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/Persona-mount_base.html">1751</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/Persona-mount.html">1725</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/SpinButton-mount_base.html">1436</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/SpinButton-mount.html">1364</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/SwatchPicker-mount_base.html">1519</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31089/perf-test-react-components/SwatchPicker-mount.html">1527</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Review by GeoffCoxMSFT — APPROVED** [review]

File: packages/react-components/react-motions-preview/stories/CreateMotionComponent/CreateMotionComponentDefault.stories.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ const useClasses = makeStyles({
   },
   item: {
     backgroundColor: tokens.colorBrandBackground,
+    ...shorthands.border('3px', 'solid', tokens.colorTransparentStroke),
```
Comment (spmonahan) [inline packages/react-components/react-motions-preview/stories/CreateMotionComponent/CreateMotionComponentDefault.stories.tsx:19]: nit: why not a token for the width as well?

```suggestion
    ...shorthands.border(tokens.strokeWidthThicker, 'solid', tokens.colorTransparentStroke),
```

**Review by spmonahan — APPROVED** [review]

File: packages/react-components/react-motions-preview/stories/CreateMotionComponent/CreateMotionComponentDefault.stories.tsx (hunk 1 above)
Comment (smhigley) [reply packages/react-components/react-motions-preview/stories/CreateMotionComponent/CreateMotionComponentDefault.stories.tsx]: Done! I ended up cleaning up some other 3px border-width defs in the same package's stories too

**Review by khmakoto — APPROVED** [review]

**Review by layershifter — APPROVED** [review]

**Pull request merged**